packages/desktop-client/src/components/reports/graphs/CalendarGraph.tsx

```
@@ -0,0 +1,294 @@
+export function CalendarGraph({
+  data,
+  start,
+  firstDayOfWeekIdx,
+  //onFilter,
+  onDayClick,
+}: CalendarGraphProps) {
+  const { t } = useTranslation();
+  const startingDate = startOfWeek(new Date(), {
+    weekStartsOn:
+      firstDayOfWeekIdx !== undefined &&
+      !isNaN(parseInt(firstDayOfWeekIdx)) &&
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)